tests/helpers.js

```diff
@@ -99,16 +115,45 @@ export const buildResultCell = ({name, keyType, testTitle}) => ({
   columnId: `${name}: ${keyType}`, rowId: testTitle
 });
 
+function getMochaTestDefinitionFromContext(testContext) {
+  if('currentTest' in testContext) {
+    // Test when called from a `beforeEach` or `afterEach` hook
+    return testContext.currentTest;
+  }
+
+  if('test' in testContext) {
+    // Test when called from a test function directly
+    return testContext.test;
+  }
+
+  throw new Error('Could not find test definition in test context');
+}
+
 export function annotateReportableTest(testContext, {
   implementationName, keyType
 }) {
-  testContext.test.cell = buildResultCell({
+  // The precise test definition object depends on the
+  // context in which this helper is called.
+  const context = getMochaTestDefinitionFromContext(testContext);
```

Comment:
I do whish we had a different word for this besides "context"... I know it's correct and in use in Mocha...but "context" (as in `@context`) is frequently used in these test suites...so...if you see an easy name change, that'd likely be preferred. If not...now worries...we'll read things [wait for it] contextually. 😅

Reply:
Hmmm maybe “definition”? I’ll chatGPT some alternatives :D

Reply:
I can just drop the “fromcontext” part since it is implied

Reply:
I think the variable name here is the problem. context is a really loaded term when working with jsonLd so perhaps `mochaContext` would help clarify the issue. If is possible to use `environment` over `context` that would work unless `context` is the official mocha term here.